stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1166,3 +1166,20 @@ public func _GetGlobalEagerContext() -> CTFEContext {
   return _ExecutionContext.global.eagerContext
 }
 
+// TODO: replace these functions with generic ones that do not hard-code Float.
```

Comment:
I think this will be more straight-forward to do when there is a non-generic superclass of TensorHandle.

Reply:
Ack. Will refresh this once I use Richard's patch.